data/json/items/comestibles.json

```diff
@@ -15196,6 +15196,93 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "milk_curdled",
+        "name" : "curdled milk",
+        "weight" : 236,
+        "color" : "white",
+        "addiction_type" : "none",
+        "spoils_in" : 672,
+        "stim" : 0,
+        "container" : "waterskin",
+        "to_hit" : 0,
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 8,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 4,
+        "description" : "Milk that has been curdled with vinegar and rennet.  It still needs to be salted and drained of whey.",
+        "price" : 10,
+        "material" : "milk",
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "liquid",
+        "charges" : 6,
```

Comment:
That charges: 6 also makes it stack way better than milk. You could craft 18 units of curdling milk from a gallon of milk and then fit it all in a plastic bottle+glass flask.